docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -81,74 +124,83 @@ After adding to the workspace, you still need to invite the bot to the channel.
 and inviting them to the channel.
 ![Invite bot to channel](../../../img/enterprise/plugins/slack/invite-user-to-team.png)
 
-## Installing the Teleport Slack plugin
+## Configuration file
 
-We recommend installing the Teleport Plugins alongside the Teleport Proxy. This is an ideal
-location as plugins have a low memory footprint, and will require both public internet access
-and Teleport Auth access.  We currently only provide linux-amd64 binaries, you can also
-compile these plugins from [source](https://github.com/gravitational/teleport-plugins/tree/master/access/slack).
+Teleport Slack uses a config file in TOML format. Generate a boilerplate config by
+running the following command and place it in the `/etc` directory where it can be
+read by the plugin.
 
-**Install the plugin**
+```code
+$ teleport-slack configure > teleport-slack.toml
+$ sudo mv teleport-slack.toml /etc
+```
 
-<Tabs>
-<TabItem label="Download">
-  ```code
-  $ curl -L https://get.gravitational.com/teleport-access-slack-v(=teleport.version=)-linux-amd64-bin.tar.gz
-  $ tar -xzf teleport-access-slack-v(=teleport.version=)-linux-amd64-bin.tar.gz
-  $ cd teleport-access-slack
-  $ ./install
-  ```
-</TabItem>
-<TabItem label="From Source">
-  To install from source you need `git` and `go >= (=teleport.golang=)` installed.
+This should result in a config file like this:
 
-  ```code
-  # Checkout teleport-plugins
-  $ git clone https://github.com/gravitational/teleport-plugins.git
-  $ cd teleport-plugins/access/slack
-  $ make
-  ```
-</TabItem>
-</Tabs>
+(!examples/resources/plugins/teleport-slack-configure.toml)
 
+#### Editing the config file
 
-Run `./install` from `teleport-slack` or place the executable in the appropriate `/usr/bin` or `/usr/local/bin` on the server installation.
+We need to edit the config files with values specific to your Teleport and Slack App intances.
 
-### Configuring the Teleport Slack plugin
+Open the config file we just generated - `/etc/teleport-slack.toml` - and you should see the following fields which we need to edit.
 
-Teleport Slack uses a config file in TOML format. Generate a boilerplate config by
-running the following command:
+**`[teleport]`**
```

Comment:
I think we should put all of the text under the `[teleport]` config option under a `Tabs` component so self-hosted/Cloud users don't see scope-irrelevant information. Currently, readers need to determine which information applies to them, and self-hosted users have to skip over the "Cloud Proxy Address" Admonition. Using one `Tabs` box for everything in the `[teleport]` discussion will make this simpler.